docs/5-operator-manual/0-set-up-and-install.md

```diff
@@ -35,23 +32,20 @@ Depending on the permissions of your user, if you are installing k3s through the
-
+Zarf deploys to almost any cluster, you are not tied down to the K3s cluster that the [init package](../4-user-guide/2-zarf-packages/3-the-zarf-init-package.md) provides! You could use local dockerized k8s cluster such as [k3d](https://k3d.io/v5.4.1/) or [KinD](https://kind.sigs.k8s.io/), Rancher's next-generation k8s distribution [RKE2](https://docs.rke2.io/), or cloud-provided clusters such as [eks](https://aws.amazon.com/eks/)
 
 # Initializing a Cluster
 <!-- TODO: Some duplicated information from the 'Common CLI Uses' page incoming... -->
 
-Now that you have the CLI installed and a cluster to work with, let's get that cluster initialized so you can deploy application packages onto it! 
+Now that you have the CLI installed and a cluster to work with, let's get that cluster initialized so you can deploy application packages onto it!
 
-Initializing a cluster is necessary since almost all k8 clusters do not come with a container registry by default. This becomes an interesting 'chicken or the egg' problem since you don't have a container registry available to push your container registry image into. This problem is discussed more in the [init package](./../4-user-guide/2-zarf-packages/3-the-zarf-init-package.md) page. 
+Initializing a cluster is necessary since almost all k8 clusters do not come with a container registry by default. This becomes an interesting 'chicken or the egg' problem since you don't have a container registry available to push your container registry image into. This problem is discussed more in the [init package](./../4-user-guide/2-zarf-packages/3-the-zarf-init-package.md) page.
 
 During the initialization process, Zarf will create a 'zarf' namespace and deploy an in-cluster Docker registry (used to host container images future packages will need), a 'zarf agent' mutating webhook (to redirect outgoing requests into the internally hosted resources), an optional gitea server (to host the git repositories future packages will need), and a handful of secrets.
 
 You can find the relevant [init package release](https://github.com/defenseunicorns/zarf/releases) on the GitHub releases page. Once downloaded, you can install the init package by navigating to the directory containing the init package and running the command [zarf init](../4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_init.md). Zarf will prompt you, asking if you want to deploy the optional component, you can type `y` or `n` depending on your use case and needs.
 
-Once the init command is finished, you can run `kubectl get pods -n zarf` to verify that the pods have come up healthy. You should expect to see two `agent-hook` pods,  a `zarf-docker-registry` pod, and optionally a `zarf-gitea` pod.
-
+Once the init command is finished, you can run `kubectl get pods -n zarf` to verify that the pods have come up healthy. You should expect to see two `agent-hook` pods, a `zarf-docker-registry` pod, and optionally a `zarf-gitea` pod.
```

Comment:
Do we want to use `zarf tools monitor` instead?

Reply:
I could see this one going either way. Doing a kubectl command is straight forward since using k9s can be annoying/difficult for someone who doesn't know how to traverse through a vim-esque terminal

Reply:
Yeah and just confirmed that it does ship with kind from brew install.  I think this is probably fine for now then...